src/Geometry/coordinates.jl

```diff
@@ -64,8 +64,13 @@ Base.show(io::IO, point::Abstract3DPoint) = print(
 Define a subtype `name` of `AbstractPoint` with appropriate conversion functions.
 """
 macro pointtype(name, fields...)
+    tofloat = :()
```

Comment:
I know it's an extra line, but can we move this into the elseif-blocks below? And, for clarity, rename the expression to something other than the method. E.g., `tofloat_expr = :()`.

I'm fine with this PR after these two changes.

Reply:
Changed!